src/microbit/model/constants.py

```diff
@@ -0,0 +1,16 @@
+BLANK = "00000:00000:00000:00000:00000:"
+BOAT = "05050:05050:05050:99999:09990:"
+BRIGHTNESS_ERR = "brightness out of bounds"
+COPY_ERR_MESSAGE = "please copy() first"
+
+HEART = "09090:99999:99999:09990:00900:"
+INCORR_IMAGE_SIZE = "image data is incorrect size"
+INDEX_ERR = "index out of bounds"

```

Comment:
have all the error messages in one section

Kind of like:
# Errors
INCORR_IMAGE_SIZE = "image data is incorrect size"
INDEX_ERR = "index out of bounds"
etc

Reply:
corrected in next commit!